logging/bin/deploy_elasticsearch_open.sh

```diff
@@ -117,6 +118,34 @@ if [ "$(helm -n $LOG_NS list --filter 'odfe' -q)" == "odfe" ]; then
    log_debug "A Helm release [odfe] exists; upgrading the release."
    existingODFE="true"
 
+   #Migrate Kibana content if upgrading from ODFE 1.7.0 to 1.13.2
+   if [ "$(helm -n logging list -o yaml --filter odfe |grep app_version)" == "- app_version: 1.8.0" ]; then
+
+      # Prior to 1.1.0 we used ODFE 1.7.0
+      log_info "Migrating from Open Distro for Elasticsearch 1.7.0"
+
+      #export exisiting content from global tenant
+      #KB_GLOBAL_EXPORT_FILE="$TMP_DIR/kibana_global_content.ndjson"
```

Comment:
The file should (if things go right) contain the exported Kibana objects.  But, if things fail, it contains potentially helpful error information.  I suppose a failed migration means the file will be created (signaling a migration)  even if the content migration step fails.  That would signal the migration import processing in deploy_kibana_content.sh to execute...and fail.  Hmmm.  But I think that's what we would want: the existence of the file indicates a migration is taking place...which we want to react to even if we can't migrate the content since there is other processing that needs to happen during a migration (other than migrating content).  At this point, I don't think a failure to export existing content should stop the migration process.  We don't really support an alternative.  I suppose we could abort and force the user to roll back to a previous release but I'm not sure that is needed at this point (when most/all of the existing Kibana content is probably our default stuff anyway).

Reply:
I was just more concerned about the export and troubleshooting data getting lost since we delete TMP_DIR by default.